storybook/pages/ExemptionsComponent/ExemptionsComponentControls.qml

```diff
@@ -0,0 +1,187 @@
+import QtQuick 2.14
```

Comment:
Please check `Exceptions PR` https://github.com/status-im/status-desktop/pull/7907# where I added some comments related to this component!